KP5-Topic2/KP4-Topic2/target/m2e-wtp/web-resources/META-INF/maven/com.aguirre/KP4-Topic2/pom.properties

```diff
@@ -0,0 +1,7 @@
+#Generated by Maven Integration for Eclipse

```

Comment:
Please include the folder target in .gitignore file to avoid to upload to the repo these generated files